.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @topos-network/protocol @topos-network/tools
```

Comment:
Hey @sebastiendan , seems `topos-network/protocol` and `topos-network/tools` don't exist yet.
<img width="800" alt="Screenshot 2023-03-28 at 1 52 27 PM" src="https://user-images.githubusercontent.com/11741977/228312216-2e8cc64a-54a5-44e3-8f0e-a5372b212e0c.png">

Reply:
Ah, they do exist but the explicit permission attribution was missing. Just gave `write` permission to the repo to the `topos-network/toposware` team (parent team of the two listed)